Compiler/Script/CevalScript.mo

```diff
@@ -1393,7 +1393,7 @@ algorithm
           if (0 == System.systemCall("unzip -q -o -d \"" + workdir + "\" \"" +  filename + "\"")) then
             b = true;
             s1 = System.basename(filename);
-            s2::_ = System.strtok(s1, ".");
+            s2 = Util.removeLast3Char(s1);
```

Comment:
Is it safe to remove a fixed amount of chars from the end???

Reply:
Yeah since I expect the filename with a `.mol` extension but I think its still a good idea to check the length of the string.